src/vm/exceptionhandling.cpp

```diff
@@ -4635,33 +4635,43 @@ VOID DECLSPEC_NORETURN UnwindManagedExceptionPass1(PAL_SEHException& ex, CONTEXT
     EEPOLICY_HANDLE_FATAL_ERROR(COR_E_EXECUTIONENGINE);
 }
 
-VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex)
+VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException)
```

Comment:
I have figured out a simplification - if we changed the SEHProcessException in src/pal/src/exception/seh.cpp so that we add 'pointers->ContextRecord.ContextFlags |= CONTEXT_EXCEPTION_ACTIVE' to the beginning of the function, then we don't have to add the isHardwareException parameter anywhere. In the code below, we would replace 'if (isHardwareException)' by 'if (ex.ContextRecord.ContextFlags & CONTEXT_EXCEPTION_ACTIVE)' and remove the setting of the flag.
Could you please change it this way?
 Also, the source code alignment is broken below - you seem to have tabs at the beginning of some lines.